test/Microsoft.AspNet.Authentication.Test/Cookies/CookieMiddlewareTests.cs

```diff
@@ -421,6 +421,27 @@ private Task SignOutAsWrong(HttpContext context)
         }
 
         [Fact]
+        public async Task CookieTurns401ToAccessDeniedWhenSetAndIfAuthenticated()
+        {
+            var clock = new TestClock();
+            TestServer server = CreateServer(options =>
+            {
+                options.SystemClock = clock;
+                options.AccessDeniedPath = new PathString("/accessdenied");
```

Comment:
Can we also add a test for the default value of `AccessDeniedPath`. Client can check if it received a `403` status code.